bigquery/google/cloud/bigquery/client.py

```diff
@@ -739,7 +739,8 @@ def list_rows(self, table, selected_fields=None, max_results=None,
 
         params = {}
         if selected_fields is not None:
-            params['selectedFields'] = [f.name for f in selected_fields]
+            params['selectedFields'] = ','.join(
+                [f.name for f in selected_fields])
```

Comment:
You don't need the square brackets (i.e. use a generator instead of a list).

Reply:
Done.